### ISSUE TITLE
Facilitate standard fonts for translating

### ISSUE THREAD

**Issue by BrokenEagle** [danbooru/danbooru#4212]
Discussed somewhat in [topic #15327](https://danbooru.donmai.us/forum_topics/15327).

Currently, picking a font while translating is pretty hit or miss, especially with smartphones. This can drastically affect the presentation of the note to the user, compared to how the translator intended it to be seen. Instead, site-wide fonts can be supported using the `@font-face` CSS rule.

https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face

This doesn't need to be an all-inclusive list of fonts, but at least several of the more common fonts like:

- Arial (sans-serif)
- Verdana (sans-serif)
    - standard font used on the site
- Helvetica (sans-serif)
    - standard font used on the site
- Times New Roman (serif)
- Comic Sans MS (cursive)

Perhaps also checking all of the current notes for instances of `font-family` to see what fonts the translators like to use.

**Comment by evazion** [comment]
I added a fallback font for Comic Sans. The other basic fonts (Arial / Verdana / Times New Roman) are already supported on most systems, either by the actual font itself or by a builtin equivalent (i.e. Android uses Noto Serif as a substitute for Times New Roman and Roboto as a substitute for Verdana). I'm using http://fontfamily.io to get this info.

The big problem here is that most fonts used in notes are copyrighted. We can't distribute the font files ourselves without buying commercial licenses, which aren't that cheap. So for every font we want to support we have to find a free alternative that's close enough to the real thing.

**Comment by BrokenEagle** [comment]
As discussed in [topic #16435](https://danbooru.donmai.us/forum_topics/16435), another suggestion for a font which meets the need for a Spooky font was suggested. I'm just adding the conversation here so that it gets documented.

> Zurreak said:
> 
> I have two potential candidates for the scary font.
> 
> The first is Anarchy. It's 100% free. It contains uppercase, lowercase, and basic punctuation, it does not contain a hyphen, accented letters, or special characters.
It has a good capital I, is easy to read, and is stylistically quite similar to the source's style on a lot of the posts I've used Chiller on in the past. I think it would make a good addition to the site.
> 
> The second is Frotten. It's 100% free. It contains uppercase, lowercase, basic punctuation, accented letters, and a variety of special characters.
It has a bad capital I, I think it's overstylized, and it most likely would rarely ever fit the source's style. On top of that it's not visually appealing and might be hard to read. I don't particularly like this font, but as one of the few passable 100% free "horror" fonts with a look similar to what we're going for I figured I might as well mention it.

I mentioned that out of those, I'd go with the Anarchy font, the same as the suggester Zurreak.
